code/game/area/areas/ruins/sandplanet.dm

```diff
@@ -5,3 +5,7 @@
 /area/ruin/whitesands/saloon
 	name = "Hermit Saloon"
 	icon_state = "green"
+
+/area/ruin/whitesands/pubbycrash
+	name = "Pubby Crash"
```

Comment:
Not required but I'd personally change this to "Crashed Pubby" or "Crashed Ship" as it will show up on the crew monitor if someone is in there.

Reply:
Im going to choose to leave it as be as i think crashed ship is very too generic to use as a area name